google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -221,4 +225,36 @@ void pollForChanges(@NotNull final CloudDebugProcessState state) {
     }
   }
 
+  private void handleBreakpointQueryError(@NotNull CloudDebugProcessState state, @NotNull Exception e) {
+    String message;
+    String projectName = state.getProject().getName();
+    if (e instanceof GoogleJsonResponseException) {
+      GoogleJsonResponseException jsonResponseException = (GoogleJsonResponseException) e;
+      if (jsonResponseException.getStatusCode() == HttpStatusCodes.STATUS_CODE_FORBIDDEN
+          || jsonResponseException.getStatusCode() == HttpStatusCodes.STATUS_CODE_UNAUTHORIZED) {
+        message = GctBundle.message("clouddebug.background.listener.access.error.message",
+                                    projectName);
+      } else {
+        message = GctBundle.message("clouddebug.background.listener.general.error.message",
+                                    projectName,
+                                    jsonResponseException.getDetails().getMessage());
+      }
+    } else {
+      message = GctBundle.message("clouddebug.background.listener.general.error.message",
+                                  projectName,
+                                  e.getLocalizedMessage());
+    }
+    handleBreakpointQueryError(state, message);
+  }
+
+  private void handleBreakpointQueryError(@NotNull CloudDebugProcessState state, String message) {
+    state.setListenInBackground(false);
+    String title = GctBundle.message("clouddebug.background.listener.error.title");
+    Notification notification =
+        new Notification(CLOUD_DEBUGGER_ERROR_NOTIFICATIONS_DISPLAY_GROUP,
+                         title,
+                         message,
+                         NotificationType.ERROR);
+    Notifications.Bus.notify(notification,  state.getProject());
```

Comment:
nit: extra space after `notification,  `
Is this simliar to the bubble message that pops up when an ide exception happens?